simple/rdm_deferred_wq.c

```diff
@@ -507,7 +507,7 @@ static int run_test()
 	}
 	printf(" with counters..."); 
 
-	init_buf_vals(tested_op);
+	init_buf_vals();
```

Comment:
agreed - I merged this since it fixes an immediate failure in travis, but this is a good cleanup change.